src/app/common/modals/progress-modal/progress-modal.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="progress-modal">
+    <div class="modal-header clearfix">
+      <h3 class="modal-title">
+        <i class="fa fa-spinner fa-2x pull-left"></i>
+        <div>{{ data.title }}</div>
+        <small>{{ data.message }}</small>
+      </h3>
+    </div>
+    <div class="modal-body">
+      <div class="large-notice-block">
+        <i class="fa fa-3x fa-spinner fa-pulse"></i>
+        <p>Processing...</p>
+      </div>
+    </div>
+    <div class="modal-footer clearfix">
+      <p class="help-block col-sm-6 col-sm-offset-3 text-center">
+        You can close this modal and you will be alerted when processing has finished
+      </p>
```

Comment:
It's good to see the use of placeholders for input fields, but providing some default text or examples as placeholders could improve usability.